exercises/concept/fibonacci-iterator/.meta/exemplar.jl

```diff
@@ -1,4 +1,9 @@
 # Step 1
+"""
+    Fib
+
+TODO (see PR comments)
+"""
```

Comment:
I wonder if this should be wrapped in some function like `genfibs`. It's quite rare that I iterate over a type instead of some function that returns an iterable but I don't know what the best practice is.

Any thoughts?

Reply:
It certainly is a bit unusual to explicitly do `for x in Fib(10)` tho we do of course iterate over values of a type all the time (vectors, dicts, etc).

I think `eachfib` and `EachFib` would be iterator-y names?